### ISSUE TITLE
Suggestion for adding missing episodes or a movie

### ISSUE THREAD

**Issue by kolbdog3333** [theMK2k/Media-Hoarder#52]
I was thinking maybe implement yt-dlp then once a person finds the video link for video of movie or episode they want to download if its supported by that they can push download after choosing the folder tv show or movie and it downloads in best quality basically just the default. Then once video is downloaded you put in the imdb id of episode or movie then it renames it to correct formatting using "%T (%Y).ext for for movie. If show almost same formatting but %N - %S%E - %T.ext for show.
 Basically T stands for Title
Y = Year
N = Episode Name
S = Season
E = Episode Name

**Comment by kolbdog3333** [comment]
This way legal since everyone uses yt-dlp.

**Comment by theMK2k** [comment]
Just because a lot of people use something doesn't make it legal.

Integrating a video downloader is interesting, I'll keep this issue open for now, but will rename the title.

**Comment by kolbdog323** [comment]
Am just wondering how the progress for the new release coming along

**Comment by theMK2k** [comment]
I plan to release it on Monday, 23rd September (the 2nd birthday of Media Hoarder)

**Comment by kolbdog323** [comment]
> I plan to release it on Monday, 23rd September (the 2nd birthday of Media Hoarder)

Thank you I wasnt sure thanks for letting me know
